changelogs/emissions-api-update-sites-filtering-12-12-2023.md

```diff
@@ -9,65 +9,65 @@ Below is a list of all the new endpoints:
 
```

Comment:
for the second 3 line type: added

Can we change to deprecated or removed? 
Could we just remove this md file since you are creating emissions-api-deprecation-v1.md

Reply:
I am not sure, changelog are suppose to be there to see changes over time, but the links are tested somewhere else so it was throwing errors.